apps/src/templates/instructions/HeightResizer.jsx

```diff
@@ -86,9 +87,9 @@ var HeightResizer = React.createClass({
   },
 
   render: function () {
-    var mainStyle = _.assign({}, styles.main, {
-      top: this.props.position
-    });
+    var mainStyle = [styles.main, {
+      top: this.props.position - RESIZER_HEIGHT
+    }];
```

Comment:
For cases where we're combining a bunch of static styles (like in MarkdownInstructions) the array of objects pattern supported by Radium seems clearly better than doing _.assign. In the case where we have dynamic styles, and thus end up creating objects on the fly, it's less clearly better, but I think I still prefer this approach as it's less verbose. Thoughts?